source/addonHandler/__init__.py

```diff
@@ -175,7 +175,29 @@ def initialize():
 
 def terminate():
 	""" Terminates the add-ons subsystem. """
-	pass
+	emptyTempDocDir()
+
+
+def getTempDocDirPath():
+	return os.path.join(tempfile.gettempdir(), "nvdaDoc")
+
+
+def getTempDocDir():

```

Comment:
```suggestion
def createTempDocDir():

```

Reply:
I am a bit concerned by `createTempDocDir` since the directory is not created if it already exists. And `createTempDocDirIfItDoesNoYetExist` seems a bit long!